### ISSUE TITLE
Missing variable when local directory is missing

### ISSUE THREAD

**Issue by christophhagen** [cytopia/linux-timemachine#77]
The variable `dir_part` is not set when local directory is missing.

https://github.com/cytopia/linux-timemachine/blob/2fe23f3402404edaa947a27811a72b0d97fcfb3a/timemachine#L184

It's only defined when using a remote directory:

https://github.com/cytopia/linux-timemachine/blob/2fe23f3402404edaa947a27811a72b0d97fcfb3a/timemachine#L166-L169

**Comment by cytopia** [comment]
@christophhagen thanks for the issue. Can you give me a run command that triggers this? Also is something not working or do you see any error message?

**Comment by christophhagen** [comment]
I was just using a simple command where the destination directory had not been created yet:

```bash
timemachine /data /missing/dir/
```

The output was:
```
/usr/local/bin/timemachine: 184: dir_part: parameter not set
```

**Comment by christophhagen** [comment]
I guess it jumps into the error case when checking that the destination exists, and then can't print the error log due to the missing variable `dir_part`

**Comment by cytopia** [comment]
Got it. Looks like this was a type and `directory` is the correct var name. PR on the way: https://github.com/cytopia/linux-timemachine/pull/78

Thanks again for reporting!

**Comment by cytopia** [comment]
Should now be fixed here:

https://github.com/cytopia/linux-timemachine/releases/tag/v1.3.2

**Comment by h0ysala** [comment]
Nice to see it's fixed now! Issue #74 relates to this.
